application/server/models/user.go

```diff
@@ -6,6 +6,7 @@ import (
 
 type User struct {
 	gorm.Model
+	ID 		 int
```

Comment:
This ID attribute is unnecessary as the gorm.Model line above it injects an ID attribute into the user struct